### PR TITLE
LJS-122 | Lattice Map package Storybook upgraded to version 6

### PR DESCRIPTION
**Pull request by ShiviStar**
In this PR, Lattice Map package Storybook has been upgraded from version 5 to 6.

Kindly Review,
Shivani Sharma

### CODE REVIEW

**Comment by Rahulgarg30591** [comment]
@ShiviStar kindly remove all the other stories which are provided by initializing the storybook and moreover please remove this file: packages/.DS_Store

File: packages/map/package.json [hunk 1]
```diff
@@ -35,21 +36,27 @@
     "@babel/plugin-transform-runtime": "^7.10.1",
     "@babel/preset-env": "^7.10.2",
     "@babel/preset-react": "^7.10.1",
-    "@latticejs/lattice-scripts": "^1.0.1-beta.2",
+    "@latticejs/lattice-scripts": "file:../lattice-scripts",
```
Comment (Rahulgarg30591) [inline packages/map/package.json]: Use the release version.

File: packages/map/src/stories/Introduction.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,194 @@
+import { Meta } from '@storybook/addon-docs/blocks';
```
Comment (Rahulgarg30591) [inline packages/map/src/stories/Introduction.stories.mdx]: This file can be deleted.

File: packages/map/test/enzyme.test.js [hunk 3]
```diff
@@ -29,40 +29,4 @@ describe('To test the Map Box Component.', () => {
   it('test <Map /> Component render', () => {
     expect(wrapper.find(Map).length).toBe(1);
   });
```
Comment (Rahulgarg30591) [inline packages/map/test/enzyme.test.js:33]: Please write the test cases of other functions that are now in use after switching to hooks.

**Review by Rahulgarg30591 — CHANGES_REQUESTED** [review]

**Comment by ShiviStar** [comment]
@Rahulgarg30591 I am not able to delete .DS_Store file as I am not able to see it.

File: packages/map/src/stories/Introduction.stories.mdx (hunk 2 above)
Comment (ShiviStar) [reply packages/map/src/stories/Introduction.stories.mdx]: We are using this file so why to delete? @Rahulgarg30591

File: packages/map/package.json (hunk 1 above)
Comment (ShiviStar) [reply packages/map/package.json]: Done

File: packages/map/src/stories/Introduction.stories.mdx (hunk 2 above)
Comment (Rahulgarg30591) [reply packages/map/src/stories/Introduction.stories.mdx]: Why and where exactly it is being used? I am unable to find any Lattice component inside the MDX file.

Comment (ShiviStar) [reply packages/map/src/stories/Introduction.stories.mdx]: Deleted the whole file as it is of no use

**Comment by ShiviStar** [comment]
@Rahulgarg30591 I have made the required changes as request by you. Please have a look and review and merge.

**Review by Rahulgarg30591 — APPROVED** [review]

**Pull request merged**